src/main/java/lesson4/ApacheWorker.java

```diff
@@ -0,0 +1,66 @@
+package lesson4;
+
+import org.apache.http.*;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.impl.client.HttpClientBuilder;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Base64;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ApacheWorker {
+
+    public static List<String> getListOfRepoUsers() throws Exception {
+
+        String owner = "namellez";
+        String repo = "CapsilonAutomation";
+        String url = "https://api.github.com/repos/" + owner + "/" + repo + "/stats/contributors";  //API: repos/:owner/:repo/stats/contributors
+
+        HttpClient client = HttpClientBuilder.create().build();
```

Comment:
is this http client differs from the one created in line 54?

Reply:
created a common static class field